deps/libmediasoupclient/src/PeerConnection.cpp

```diff
@@ -129,19 +129,22 @@ namespace mediasoupclient
 	{
 		MSC_TRACE();
 
-		webrtc::RTCError error = this->pc->SetConfiguration(config);
```

Comment:
Idem here. Are you sure you are linking against libwebrtc m79 branch? This should only be needed from m81 onwards. I have not checked m80 but is not needed in m79 for sure, which is the branch used by `libmediasoupclient` v3. The branch used in this demo..

Reply:
I'm using libwebrtc m74 branch. After reading [this](https://mediasoup.org/documentation/v3/libmediasoupclient/installation/), I thought m74 is the branch I should be using with any libmediasoupclient (v3 branch) code (so including this project on v3 branch).

TBH now I'm a bit confused ... let me know, I can build it also with libwebrtc m79.

Reply:
Just to be clear, please let me know which branch of mediasoup-broadcaster-demo should I build using which branch of libwebrtc.

That is: 
A. v3 + libwebrtc m74
B. v3 + libwebrtc m79
C. m79 + libwebrtc m79
D. all the above
E. something else

Reply:
v3 branch in libmediasoupclient and v3 branch in mediasoup-broadcaster-demo. In addition, v3 in libmediasoupclient uses m79 and hence mediasoup-broadcaster-demo must also use m79.